lib/src/services/gamepad.dart

```diff
@@ -0,0 +1,42 @@
+import "package:win32_gamepad/win32_gamepad.dart";
+
+import "service.dart";
+
+///A service to send the comtroller's activity
+///using win32_gamepad
+class GamepadService extends Service {
+  ///The connection to the controller
+  static final gamepad = Gamepad(0);
+  
+  @override
+  Future<void> init() async{ 
+    connect();
+  }
+
+  @override
+  Future<void> dispose() async{  
+    gamepad.state = GamepadState.disconnected();
+  } 
+
+  ///Check to see if the remote is connected
+  ///
+  ///returns true or false
+  Future<bool> connect() async{ 
+    if(gamepad.isConnected){
+      return true;
+    }else{
+      return false;
+    }
+  }
```

Comment:
```suggestion
/// Whether the gamepad is connected to the user's device. 
bool get isConnected => gamepad.isConnected;
```
If there's no way to actually connect to the controller then we just shouldn't have this. If you want you can have
Seems like there's no way to actually "connect" to a controller -- it's either there or not. So this might be more appropriate.